modules/setting/mirror.go

```diff
@@ -48,11 +48,7 @@ func loadMirrorFrom(rootCfg ConfigProvider) {
 		Mirror.MinInterval = 1 * time.Minute
 	}
 	if Mirror.DefaultInterval < Mirror.MinInterval {
-		if time.Hour*8 < Mirror.MinInterval {
-			Mirror.DefaultInterval = Mirror.MinInterval
-		} else {
-			Mirror.DefaultInterval = time.Hour * 8
-		}
+		Mirror.DefaultInterval = max(time.Hour*8, Mirror.MinInterval)
```

Comment:
I agree with the tool, that's what the code is doing.
However, it's not what it **should** be doing:
I think the code is supposed to be

```suggestion
		Mirror.DefaultInterval = Mirror.MinInterval
```
Comparing against the default value looks strange

Reply:
Maybe we should keep behaviour changes out of this PR, to keep things simple 😉.

Reply:
Yes, avoid unnecessary changes.